src/com_tjvendors/site/models/vendor.php

```diff
@@ -523,6 +533,10 @@ public function save($data)
 				/* Send mail on vendor creation */
 				$tjvendorsTriggerVendor->onAfterVendorSave($data, true);
 
+				// Plugin trigger
+				PluginHelper::importPlugin('tjvendors');
+				Factory::getApplication()->triggerEvent('tjVendorOnAfterVendorSave', array());
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done